jvm/src/main/scala/com/nawforce/apexlink/rpc/OrgAPIImpl.scala

```diff
@@ -147,7 +148,13 @@ case class IssuesForFile(promise: Promise[IssuesResult], path: String) extends A
   override def process(queue: OrgQueue): Unit = {
     val orgImpl = queue.org.asInstanceOf[OPM.OrgImpl]
     OrgInfo.current.withValue(orgImpl) {
-      promise.success(IssuesResult(orgImpl.issues.issuesForFileInternal(Path(path)).toArray))
+      promise.success(
+        IssuesResult(
+          orgImpl.issues
+            .issuesForFilesInternal(Array(Path(path)), includeWarnings = true, maxIssuesPerFile = 0)
+            .toArray
```

Comment:
This looks odd out of context, I'm wondering why its changed but maybe I'll find out later.